src/Indice.Features.Cases.AspNetCore/Models/Requests/AddAccessRuleRequest.cs

```diff
@@ -19,9 +19,19 @@ public class AddAccessRuleRequest
     /// <summary>Access level for member on the resource.</summary>
     public int AccessLevel { get; set; }
 
-    public bool IsValid() {
-        return
-            (RuleCaseId != null || RuleCheckpointTypeId != null || RuleCaseTypeId != null ) &&
-            !(string.IsNullOrEmpty(MemberRole) && string.IsNullOrEmpty(MemberGroupId) && string.IsNullOrEmpty(MemberUserId));
-    }
+    public bool IsValid() => 
+            //Validate Rules
```

Comment:
επειδή είναι instance method άνετα φτιαχνουμε unit test με ολα τα πιθανα σεναρια ωστε να ξερουμε τι ελεγχουμε, και πως λειτουργεί, γιατι εδω μεσα δυσκολα βγαινεις αν μπεις σε Χ μήνες από τώρα

Reply:
This could be potentially simplified